pyuvsim/tests/test_simsetup.py

```diff
@@ -4,20 +4,20 @@
 
 from __future__ import absolute_import, division, print_function
 
-import numpy as np
+import copy
 import os
-import yaml
 import shutil
-import copy
-from six.moves import range
-import pytest
+
 import astropy
-from astropy.time import Time
-from astropy.coordinates import Angle, SkyCoord, EarthLocation, AltAz
+import numpy as np
```

Comment:
What's the new logic behind the order of import statements?

Reply:
The logic is actually in pep8, I've been trying to get it right but clearly haven't found all the cases. 

imports should be ordered by:
first standard lib imports
then related packages
then this package

I think there is a bit weirdness here in that imports from the same library are getting separated in a way that makes it hard for me to read them (e.g. pyuvdata.utils is not next to the UVBeam & UVData imports from pyuvdata).

Reply:
That's pretty much what I've seen so far. I'm just a bit confused here because it looks like the `numpy` import, arguably the _most_ standard library, is getting pushed down in the list. It looks like this happened automatically.

Reply:
Let me clarify: what is meant by "standard library" are libraries that are packaged with python.

I agree about numpy, and my approach, which differs from flake8 clearly, is to actually have 4 sections, where I insert things like numpy, yaml, six and maybe astropy into a section that comes just after the standardlib section and before the related packages, where I would put pyuvdata and maybe astropy.

Reply:
@steven-murray will flake8 error if we order things as I describe above?

Reply:
I don't think flake8 will complain -- my editor makes those PEP8 changes automatically. Flake8 will complain if an import is not at the top of the module (i.e. there's logic above it), but I don't think it really cares about ordering too much.

Reply:
Ok, then my vote is 4 sections as I described and to have imports from the same library next to each other. What do others think?

Reply:
do you want me to re-order the imports again? To be honest, I don't really see much of a difference between imports like `numpy` and `astropy` and `yaml`: to me it's fine if they're all lumped together. But if other people feel strongly, I can try moving a subset of non-standard libraries into their own little group?

Reply:
I'm ok with this as long as imports from the same library are together.